src/image-sync-manager.ts

```diff
@@ -67,14 +67,16 @@ export class ImageSyncManager {
   private _onTimedMetadata = () => {
     // TODO: use single "metadata" TextTrack once cue-point manager become use it
     const activeCuePoints: Array<Cue> = Array.from(this._mainPlayer.cuePointManager.getActiveCuePoints());
-    const {activeSlide, streamLayout} = activeCuePoints.reduce<{activeSlide: string | null; streamLayout: StreamLayout | null}>(
+    const {activeSlide, streamLayout} = activeCuePoints.reduce<{activeSlide: string | null; streamLayout: ExternalLayout | null}>(
```

Comment:
please update also param name from streamLayout to externalLayoutChange or something